### DISCUSSION TITLE
rust-course/into-rust

### DISCUSSION THREAD

**Discussion by sunface** [sunface/rust-course#629] [章节评论区]
https://course.rs/into-rust.html

**Comment by Gavin0x0** [comment]
酷！开学！

**Reply by roseduan** [reply]
硬核实战项目，Rust 手写数据库系统！

我使用 Rust 写了一个数据库系统教程，t 从零实现一个完整的 SQL 数据库系统，将会由浅到深、由易到难，循序渐进的带着大家去实现，从第一行代码开始，均有完整的代码实现演示。
通过此项目，你可以：
入门并巩固 Rust 基础，课程内容几乎涵盖了 Rust 大多数常用基础，例如数据类型，match 表达式匹配，函数，闭包，结构体，泛型，Trait，所有权，借用，生命周期，错误处理，智能指针等等。
增加 Rust 项目经验的同学，如果学习了一些 Rust 的基础知识，但是苦于没有项目经验，想要进一步巩固自己的知识，自己写一个数据库是一个很合适的实战项目。

教程地址：https://w02agegxg3.feishu.cn/docx/OxwGdeM30oss7vxEG5AcUn4unEc

**Reply by CListery** [reply]
跑这来卖课来了，真是不要碧莲

**Reply by Liangviolet** [reply]
卖课的是真的恶心，来别人这种开源教程下面发是真的不要脸，这种吃相真想喷死你🤮

**Reply by Paul-from-NJTech** [reply]
现在的培训班，我看见一个骂一个，有你们的存在，一个行业就容易遭受毁灭性打击。

**Reply by rovesoul** [reply]
woo! 非常不讨喜

**Reply by jyobinn** [reply]
真是非常不讨人喜欢 这个卖课

**Reply by mrlitao** [reply]
你在开源教程下卖课，还搞这么贵？是缺钱救命吗？

**Comment by 0oYGZo0** [comment]
群主牛逼

**Comment by Opticcss** [comment]
儘最大力量支持！！以後要開始好好學習 `Rust` 嘍 :)

**Reply by sunface** [reply]
感谢支持，加油 ：）

**Comment by mxismean** [comment]
Ok, let‘s go

**Reply by sunface** [reply]
roger that

**Reply by bingyunxl** [reply]
ggfhfgtgdtudyufhkdsfnew':l;jk
/h;kjjhtrreerrererererereterrererdre

**Reply by IlichiNiKoka** [reply]
赶紧学完回家吃红烧鸡翅啦~
红烧鸡翅~我最爱吃~

**Reply by itsgodzilla** [reply]
Team ,Full back!

**Comment by zengleyi** [comment]
作者功德无量！

**Reply by sunface** [reply]
不敢当， 不敢当🤣

**Comment by fenglea** [comment]
没学过语言的适合Rust语言作为第一种语言嘛

**Reply by fenglea** [reply]
或者哪个语言适合新手

**Reply by sunface** [reply]
你可以学学试试，学完后写一个小工具玩玩，如果两眼抹黑不知道该如何下手，挣扎一番后，还是搞不定，那最好先学门简单的语言，例如 js、python、go

**Reply by zhangdekui** [reply]
这就像玩魔方一样, 先从3阶入门, 然后逐渐升阶. 是有段位要求的.
还有, 毕竟语言只是工具, 还有数据结构, 网络, 数据库等等等, 多着呢, 慢慢来, 最后在啃rust

**Reply by zhangdekui** [reply]
建议先学python, 这个最简单, 然后golang(有指针的概念), 最后再rust

**Reply by dartft** [reply]
我觉得没问题，因为就算学了python该写不出来还是写不出来（bushi

**Reply by XiaoPengYouCode** [reply]
我觉得很有道理，没有经过项目的历练以及自己的研究思考，确实写不出代码

**Reply by LanYebo** [reply]
可以先从c++开始，因为没有踩过坑可能rust很多限制会让你摸不着头脑

**Reply by jyobinn** [reply]
学了python 很快就能起来 比java还简单 和js有点像 不 有报错 比js更好

**Reply by jyobinn** [reply]
当年学c++ 没学好 学c语言也没学号 数据结构刚刚入门  就卡在指针

**Comment by hrshy** [comment]
您好，我是人民邮电出版社的编辑，请问您有兴趣将这本开源的书真正的出版吗？希望得到您的回复，我的QQ是43457121

**Comment by lijing-2008** [comment]
看完官方文档还是感觉朦朦胧胧，希望读完圣经能通透起来！开搞！谢谢作者大大们的无私奉献！

**Comment by vimiix** [comment]
感谢您的辛勤付出和无私奉献！🌸

**Comment by PuffMeow** [comment]
支持

**Comment by lizhiyu-me** [comment]
都不跟我js/ts比一比的吗?

**Reply by yammyxing** [reply]
[Lol]

**Comment by dartft** [comment]
がんばれ！加油啊家人

**Comment by fritz1149** [comment]
花费自己的精力和时间，开源给大家写书，作者我想说你太伟大了

**Reply by sunface** [reply]
没有，我没有 :D

**Comment by dartft** [comment]
额，我不是作者。。。。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年6月19日(星期天) 晚上6:25
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [sunface/rust-course] into-rust (Discussion #629)





 
花费自己的精力和时间，开源给大家写书，作者我想说你太伟大了
 
—
‎直接回复此电子邮件、‎‎在 GitHub 上查看‎‎或‎‎取消订阅‎‎。‎
‎您收到此消息是因为您进行了评论。‎Message ID: ***@***.***&gt;

**Reply by fritz1149** [reply]
还是很厉害呀，你们的无私付出大家都看在眼里

**Comment by Kerwin-L** [comment]
看完都热血沸腾了！

**Comment by sunfu12** [comment]
不要光讲优点，也讲讲缺点，因为不可能没有缺点，让我们更全面地了解rust

**Comment by Newmanlixin108108** [comment]
非常棒的语言，我距离面试只有7天的时间，我要抓紧重新刷刷，期待有更好的结果❤️

**Comment by iszmxw** [comment]
# 开搞，干起来！

**Comment by ifmagic** [comment]
这么强，拜读

**Comment by 827551331** [comment]
加油，如果有机会，我也想 join

**Comment by yeisrr** [comment]
好耶！开学！

**Comment by YoHo-YuHu** [comment]
开整，开整

**Comment by Oxnorman** [comment]
学好然后用起来rust或许才是作者最大的期望吧，搞起来！

**Comment by zhaoyuanmeng** [comment]
学习起来

**Comment by chenqiw9** [comment]
学爆！

**Comment by fovegage** [comment]
# 开学

**Comment by sure2web3** [comment]
开始学习，记录一下

**Comment by xichaoli** [comment]
再次入门...

**Comment by Orakky** [comment]
再次入门

**Comment by hamgeorge** [comment]
开干

**Comment by Bdbmzwsc** [comment]
Orz

**Comment by tombflylee** [comment]
哈哈哈，继续卷😭

**Comment by WeixinX** [comment]
鸡汤打满，一库走！

**Comment by Heilum** [comment]
Thanks!

**Comment by adomit** [comment]
已阅

**Comment by just4id** [comment]
小的们，给我拿下Rust!

**Comment by ttxs25830** [comment]
看完回来看的，感觉这部分好像吹的略微过分，说实话我看了一些油管上的视频，基本没有人提到Rust熟悉后反而开发效率更高的情况，有没有同时精通另一门语言的Rust大神来说说自己的感受？

另外补充两个个人感觉的Rust的优点
1.开创了许多好语法，首先是rust独有的枚举类型，应该说是结合了C(pp)里的共用体和枚举，似乎是独一家的。其次是所有语句块末尾返回值，省去了好多一次性lamada
2.Rust目前地位很特殊，C和C++没有完善的工具链，有些问题难以集中解决，golang作为GC语言又天生不适合某些工作，所以在如WASM领域Rust一马当先

**Comment by bobo-rs** [comment]
12月份刚学完《Rust 程序设计语言》，准备用Rust写一个处理PSD图片的项目，建好项目之后，就不会写代码了，不知如何下手，希望继续学完该教程，能够写一个简单项目，祈祷

**Comment by 8tme** [comment]
冲 我倒要看看  难到什么程度

**Comment by Abab-bk** [comment]
菜逼也要学

**Comment by LevelUpper6** [comment]
##阅

**Comment by zhouyu00** [comment]
非常用心的教程，真的是属于掰开揉碎了喂给我了

**Comment by XuCpeng** [comment]
以我浅见，我号召大家 **不要发“打卡”、“阅”等评论** ，取而代之的应该是有价值的反馈、问题和讨论。如果想表达“我看到了作者的付出”，请通过star和点赞实现。主要基于以下原因：

-  **在评论区的“打卡”无助于您进行学习记录** 。因为您无法在越来越长的评论区中找到自己的“打卡”，同样的，该书也没有一个“蓝图”可以让您分析自己的打卡频次；

- 大量的“打卡”评论，显然会 **高度稀释评论区的评论内容** ，无助于有价值的信息进行集中的讨论。例如对于书中内容的勘误、学习者的疑惑、解答者的更新；

我明白，我们的学习环境培养了一系列的诸如“打卡”、“评论”、“互动”之类的学习习惯或者说社交习惯。但我希望同志们能够认真思考该行为的真正意义，并在合适的场合使用。

**Reply by sunface** [reply]
有一定道理，哈哈

**Comment by zhuzhh** [comment]
`rust` 无 `GC`，那他是怎么做到内存回收的？
回收时也是需要全局暂停吗？

**Comment by blkcor** [comment]
感谢作者 知乎来的 很厉害

**Comment by tiandankanfeng** [comment]
Get it!

**Comment by EricChengBlockChain** [comment]
每天看这么多英文文档, 我问你, power怎么翻译?  是‘力’ 对吗?  是工地搬砖,一把扛起了10包水泥的力对吗? 还是你动脑筋能3小时解答5道ACM算法题的力? 还是你开个早会 安排5人一天活的力? 还是你说句话让别人去找人干一个项目的力? 还是你说一句话, 让一大波人去忙活的力? 
凡是要靠自己不管是体力还是脑力, 都是最次的力.  开始意识到power 的更高层级的含义才是真正的力. 一个对power有更高层次理解的初中生,能开着帕拉梅拉捏压秃顶的身体力耕的博士, 不是说博士能力不行, 是对power的理解还没上来. 梭哈抄底吧兄弟. 善良没问题, 有道德也没错, 但是你想靠搬砖就开帕拉梅拉你就过分了, 你想不冒险就想翻身, 我觉得你不公平, 你说是不是, 所以, 听我的, 一把梭哈.  正常讲, 这是唯一不用秃顶,也不用违背道德,能快速翻身的机会,对吗?

**Reply by EricChengBlockChain** [reply]
抄底茅台

**Reply by sunface** [reply]
秀～～

**Reply by panxy2013** [reply]
Yes, 机会来了. 骚年们

**Comment by bazingaedward** [comment]
ok ,lets go

**Comment by Claude-Zq** [comment]
thx

**Comment by qltang789** [comment]
写的真不错

**Comment by pangsitao** [comment]
OK LET'S RUST

**Reply by sunface** [reply]
这个好！

**Comment by mjy-coder** [comment]
牛🐮开学！

**Comment by zhenzhenChange** [comment]
Ok, Little Sunflower Mum's class is now in session!

**Comment by panxy2013** [comment]
从XR/ALVR 来到这里, Go go go~

**Comment by gyt95** [comment]
因为前端领域开始出现越来越多基于Rust的基础建设了，所以就想来学一下。本来看的是疑似官方的中文文档，后来看到这个文档还挺特别，有种感觉是很希望Rust能在中国推行出去。于是我也关注了并且加群学习。

作为用了JavaScript多年的前端（大学学的C和Java已经忘光，工作后接触过Python但没学完）一直在想会不会学个Go再学Rust好一些？还是说对Node有深刻了解后也是可以的？我有点摇摆不定，但其实最好还是Node->Go->Rust的顺序吧？但对我来说Go可能只是用来Web开发（格局小了）。。但Rust我可以争取给某些前端开源项目提PR哈哈哈（Go当然也不是不能提），反正我觉得都应该学。

害，前端也很卷。。（如果有人也是前端来了解Rust的可以点我个赞呀哈哈哈，虽然感觉前端来学Rust的微乎其微！我认识的前端大多数都只爱写页面）

BTW：相比Go的吉祥物我更喜欢Rust的吉祥物，get到它的可爱。。

**Reply by MySetsuna** [reply]
同前端，一起加油

**Comment by nihaodekuangshang** [comment]
最近的消息，微软将用rust重写windows核心代码。我对rust最大的担忧就是那个基金会，毕竟之前的那个草案太吓人了。

**Comment by xllhxd** [comment]
“语言毕竟只是工具，我们不能神话它，但是可以给它一个机会，也许，你最终能收获自己的真爱 ：)”为语言论战画上了一个句号，但语言论站何尝不是一件有趣的事😋

**Comment by abnereel** [comment]
nice

**Comment by NicoleRobin** [comment]
感谢作者，学起

**Comment by go-2021** [comment]
Ok, Let's Rust!

**Comment by Renz2018** [comment]
说cargo是最好的包管理有点言过其实，多阶段构建的Dockerfile，不支持仅拉取及编译外部依赖, 很难利用docker的缓存， 不可以在命令中指定registe只能在Dockerfile写多行丑陋输出到.cargo/config
用下来，既不如go mod也不如js的npm

**Comment by ImSven** [comment]
funny！！ ok, let`s rush

**Comment by KevinSu27** [comment]
Let's rust

**Comment by sedationh** [comment]
nice 👍

**Comment by cowcowbull** [comment]
无编程基础，能学吗？

**Reply by sunface** [reply]
不建议，Rust 不是一门好的新手语言，Javascript 多好，特别有利于提升编程成就感

**Comment by pipipapi** [comment]
perfect。   加油

**Comment by liwuhou** [comment]
我的大刀早已饥渴难耐

**Comment by DAIMENGEN4** [comment]
作者，功德无量

**Comment by DAIMENGEN4** [comment]
必须支持一波

**Reply by sunface** [reply]
谢谢哈哈

**Comment by StempG** [comment]
OK,let's rust

**Comment by ddx533534** [comment]
重新激起了我学习的欲望！

**Comment by Denver-Ning** [comment]
OK，let's Rust

**Comment by evanryuu** [comment]
作者好，首先非常感谢大佬的付出，写出如此优you雅mo的rust入门教程！

在看完这章之后，我个人有个疑问：

我目前的主语言是js/ts，如果我最终想做**图形**或者**游戏引擎**相关的事情，作者大大推荐深入rust吗？

我知道在游戏引擎这个领域里rust已经有bevy的实践了，但是我不确定是该深入c++还是rust，毕竟前者在这两方面已经有了几十年的沉淀，而深入其中任何一个可能就已经耗费我所有精力了，现在正处于一个抉择的阶段，所以可以请问下sunface大佬的意见吗？

**Reply by sunface** [reply]
我个人建议是，工作继续用 C++ ，兴趣爱好可以关注下 Rust。

一款游戏引擎的成熟需要时间太久了，这个过程你可以伴随它一起成长，毕竟深入学习一个游戏引擎对于自己的能力提升也是非常大的。但是工作，需要的是开箱即用和成熟的生态，说实话，就算 C++ 最好的开源引擎 Godot 距离 Unity 等商业引擎的差距也是巨大的。

所以，工作和爱好有的时候还是要区分一下，也许有一天他们可以合而为一，但是还不是现在。

**Reply by evanryuu** [reply]
明白了，我会好好权衡的，非常感谢飞佬！

**Comment by floiges** [comment]
学起来

**Comment by GarenJian-Intel** [comment]
"现在有不少用 Rust 重写的工具、平台都超过了原来用 C、C++ 实现的版本，将老前辈拍死在沙滩上，俨然成为一种潮流～～"
这句话，可以举几个例子吗？我想去测一下性能。因为对于我们来说就算是3%的性能差异可能是优化很久而来的。如果迁移到Rust，不希望有任何性能损失。

**Reply by baifen** [reply]
ripgrep vs grep

**Comment by VisZhangrong** [comment]
我一个前端来学

**Comment by ghost** [comment]
酷！开学！+1

**Comment by wanjm** [comment]
哇，找到组织了。

**Comment by lihaohaodegit** [comment]
怎么只有跟其他语言对比的优势，没有劣势

**Comment by Leopold-Fitz-AI** [comment]
OK，加入学习

**Comment by hongspell** [comment]
整！

**Comment by RJHD-GIT** [comment]
前程似锦，未来可期

**Comment by Pudge-tao** [comment]
mark，打卡

**Comment by C-Jeril** [comment]
整

**Comment by echoLC** [comment]
二刷，重新开学

**Comment by pitlst** [comment]
开学！

**Comment by jianxchen** [comment]
so cool ! 开学

**Comment by Monkey-Z** [comment]
更适合中国宝宝的优秀书籍~

**Comment by lividsu** [comment]
OK LET'S GO

**Comment by TStoneLee** [comment]
牛逼，学起来，官方文档看的很难受

**Comment by sanrentai** [comment]
开学

**Comment by 4794-sourse** [comment]
Ok, Let's Rust！

**Comment by wuxiy** [comment]
酷！

**Comment by coderew** [comment]
2024.02.02

**Comment by lemonacy** [comment]
学一门语言最好找好使用场景，要么是工作中用到，要么是平时开发小项目用，亦或是读Linux Kernel中Rust源码用。不然学完就忘呀...

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by xingcxb** [comment]
rust 启动！

**Comment by yanyunwu** [comment]
大佬太强了😭

**Comment by NobleNoName** [comment]
从今天开始学习

**Comment by Lin123233** [comment]
有趣

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by oudef** [comment]
感谢楼主分享

**Comment by shanchuann** [comment]
开学！

**Comment by cagedbird043** [comment]
学习啦

**Comment by jianquanmao** [comment]
Thank you for your contribution. I am starting on board now.

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by xbsheng** [comment]
学~

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by lanbinshijie** [comment]
2024.05.24 开学

**Comment by yanxxit** [comment]
Ok, Let's Rust.

**Comment by Archerin** [comment]
OK，go！

**Comment by meirenhuikan** [comment]
Ok, let‘s go

**Comment by hnlisf** [comment]
Ok, let‘s go

**Comment by hnlisf** [comment]
Ok, let‘s go

**Comment by xaluoqone** [comment]
Android开发者前来瞥一眼

**Comment by fengziboboy** [comment]
第二次开始学习，希望能走远一点

**Comment by mtdxwb** [comment]
加油！！！

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Denszh** [comment]
希望能坚持下来！20240816公司午休  ML

**Comment by LiuShenXi** [comment]
从零开始的 rust 世界生活！

**Comment by Ber1276** [comment]
错别字提示：“我们不能神话它”的话，应该是神化

**Comment by crazy-cpu** [comment]
优秀优秀，必须学！最近想开发一个液晶屏显示程序，首先想到是Rust

**Comment by LeoWangJ** [comment]
2024/8/31 開學！

**Comment by inkwise** [comment]
hi

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by mason1799** [comment]
Mark

**Comment by WindAFeng** [comment]
course.rs大学入学！

**Comment by L0neW4nderer** [comment]
笨人之前一直在玩godot，国庆节之后粗略过了一遍官方book，想用wgpu做个小程序发现异常的艰难
🤓现在来开始看大佬写的文章仔细过一遍啦

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by lovexingchen** [comment]
第一次来，很期待，无论过程有多困难，我会坚持下去，希望一起看见曙光。

**Comment by Anymore0010** [comment]
第N次从头再来

**Comment by xtaens** [comment]
今天开始学习rust，因为有需要，应该不会从入门到放弃了

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by ashinewu** [comment]
mark，学习的第一天

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by utopioooo** [comment]
OK. Let's Rust!

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by rovesoul** [comment]
您的来信已经收到,谢谢。待我尽快上线处理。
                                           
                                       ——董汇标

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by rovesoul** [comment]
您的来信已经收到,谢谢。我会尽快上线处理。
                                           
                                       ——董汇标

**Comment by LinWeiJH** [comment]
这学习曲线能叫陡峭吗？开玩笑！！！叫直角！90°那种！

**Comment by Denn1sHe** [comment]
我是一个Python为主的程序员，现在python社区pyo3简直不要太火，希望通过这个教程能够学习rust的使用。感谢作者开源创作。

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by kmeykranz** [comment]
前辈写的太好了，我现在激情满满。Let's Rust!

**Comment by baoheipojis** [comment]
书中提到区块链是Rust为主？但是我认为是Solidity为主，Rust后来居上。EVM链只能用Solidity，但是近年的很多新链，sol等可以用Rust了。

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by rovesoul** [comment]
您的来信已经收到,谢谢。我会尽快上线处理。
                                           
                                       ——董汇标

**Comment by lazyclaw-labs** [comment]
有什么靠谱的中国镜像 for crates-io?

**Comment by Y-ZD-heheka** [comment]
感谢作者的辛苦付出！！！
https://github.com/Y-ZD-heheka/rust_learn
用AI做的可运行案例，圣经看过基础可以看我的项目辅助学习

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by rovesoul** [comment]
您的来信已经收到,谢谢。我会尽快上线处理。
                                           
                                       ——董汇标

**Comment by 242282218** [comment]
感谢，你的教程给你点星星了，如果找到工作那就更好了

**Comment by Appleuiy** [comment]
let's rust!

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by rovesoul** [comment]
您的来信已经收到,谢谢。我会尽快上线处理。
                                           
                                       ——董汇标

**Comment by kangcp** [comment]
牛逼，开学！

**Comment by itsgodzilla** [comment]
因吹斯听，说到最后，你还是在吹Go语言啊！你这个小Go子，CS中的经典台词是，Ok, Let's Go!

**Comment by wuwuwu-lxb** [comment]
let‘s Rust

**Comment by baifen** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by rovesoul** [comment]
您的来信已经收到,谢谢。我会尽快上线处理。
                                           
                                       ——董汇标
